### ISSUE TITLE
[request] grpc/1.59.2 or later

### ISSUE THREAD

**Issue by grmcdorman** [conan-io/conan-center-index#23036]
### Package Name/Version

grpc/1.54.3

### Changelog

https://github.com/grpc/grpc/releases/tag/v1.59.2

### Context about the new update

Versions prior to 1.59.2 are vulnerable to CVS-2023-44487 (a DoS attack, CVSS score 7.5). Version 1.59.2 contains the fix; this is not the most recent version, however, which appears to be 1.62.1 at the time of writing. Given the high CVSS score this is fairly urgent.

**Comment by Ahajha** [comment]
There's an existing PR open to add 1.60.0: https://github.com/conan-io/conan-center-index/pull/21593

It could be updated to the latest as well. Regardless, it is blocked by a PR to update protobuf here: https://github.com/conan-io/conan-center-index/pull/21593

**Comment by grmcdorman** [comment]
Silly me, I forgot to check for existing requests. My apologies for that. Will add comments to the other tickets.

**Comment by grmcdorman** [comment]
You have the same ID for the Protobuf update, perhaps you mean #22527?

**Comment by Ahajha** [comment]
Sorry about that, I meant #21622. Code-wise it should be ready, it's just missing a dependency.

**Comment by grmcdorman** [comment]
#21622 is still not ready to merge, appears to be an error with Conan v2 pipeline.

**Comment by matlec-siemens** [comment]
Now that #24154 has been merged, is there any timeline for updating gRPC? As pointed out by @grmcdorman, the latest version of gRPC in Conan Center is affected by a vulnerability with a CVSS score of 7.5 and we need an update very soon. @jcar87 @valgur

**Comment by jcar87** [comment]
Possibly by the end of the week - already done some preliminary tests, there's some issues to iron out. 
What is the earliest version that is *not* affected by the CVE?

**Comment by matlec-siemens** [comment]
> To the best of my knowledge, the earliest fix version is 1.59.2. EDIT: This is well behind the current version, though, which is 1.64.1.

Exactly, 1.59.2 is the minimum version we need, but any later version such as 1.64.1 would be fine.

**Comment by saschasc** [comment]
@jcar87 already [mentioned](https://github.com/conan-io/conan-center-index/pull/21622#issuecomment-2139056248) it before that this should also be coordinated updating the consumers like (google-cloud-cpp).  Thanks for pushing this topic. Much appreciated!

**Comment by swaeberle** [comment]
There seems to exist another attempt updating grpc and its dependencies: #23084

**Comment by lia-viam** [comment]
For anyone else following this I'm now seeing 1.65 in conan center

**Comment by grmcdorman** [comment]
> For anyone else following this I'm now seeing 1.65 in conan center

Thanks.

Interesting; at this time 'conan.io/center' still reports 1.54.3, but `conan search` sees 1.65.0. Latest grpc branch is 1.65.2; change log only says "this release contains refinements, improvements, and bug fixes" with _no_ indication for either 1.65.1 or 1.65.2 of what those are.

**Comment by jcar87** [comment]
Hi @grmcdorman , I've reported the website discrepancy here: https://github.com/conan-io/web/issues/178

As for the changelog for grpc, I'm afraid you'll have to check with the maintainers of the project - as a last resort you may be able to visualise the git diff between the tagged release commits.

**Comment by jcar87** [comment]
Closing this as resolved, since we now provide a much more recent version of gRPC. Please reopen if there if does not satisfy the request, or open a new issue if there are further issues/requests, thanks!

**Comment by grmcdorman** [comment]
> Hi @grmcdorman , I've reported the website discrepancy here: [conan-io/web#178](https://github.com/conan-io/web/issues/178)
> 

I suspected it was just a caching issue and didn't worry too much about it. I appreciate you filed a bug - as of now it's still reporting 1.54.3, I see. Will let the web UI maintainers worry about it :-)

> As for the changelog for grpc, I'm afraid you'll have to check with the maintainers of the project - as a last resort you may be able to visualise the git diff between the tagged release commits.

Yeah, I wasn't suggesting that anyone here do anything about that, just remarking that the gRPC maintainers are making our lives difficult.
